ui/imports/utils/Global.qml

```diff
@@ -40,8 +40,8 @@ QtObject {
     function openPopup(popupComponent, params = {}) {
         const popup = popupComponent.createObject(root.appMain, params);
         popup.x = Qt.binding(function() { return (root.appMain.width - popup.width) / 2; });
-        popup.y = popup.margins;
-        popup.height = Qt.binding(function() { return root.appMain.height - popup.margins * 2; });
+        popup.y = popup.topMargin;
+        popup.height = Qt.binding(function() { return root.appMain.height - popup.topMargin - popup.bottomMargin; });
```

Comment:
As discussed this part is not needed at all. See https://github.com/status-im/status-desktop/issues/6049 and the solution in comments and related PR.

These steps are needed to make popups auto-resizable:
1) `anchors.centerIn: parent`
2) `margins: 64` (already set in StatusModal)
3) **do not define explicit sizes** - otherwise qml popup will not auto-resize itself to fit window and margins

Reply:
AD. 1. I agree, `anchors.centerIn: Overlay.overlay` should be enough
AD. 2. are you sure it shouldn't shrink if the viewport becomes too narrow? I haven't said to define `implicitHeight` for popup in every scenario, because as you said, it may be ignored (overwritten) by contentItem's implicit size. Popup sizing is tricky, see the [discussion](https://github.com/status-im/status-desktop/pull/6010#discussion_r892127314).

Reply:
1. I've added `anchors.centerIn: Overlay.overlay` to StatusModal and now removing `anchors.centerIn: parent` 
2. It is not possible to make mainwindow too narrow to affect modals. In current implementation most modals designed from fixed width and it usually fixed with of whole modal. If we decide to set width in content we will need to calculate for every case manually from designs.  
3.  One more strange thing: `onClosed: destroy()` is missing in some cases, can we put it in StatusModal and limit it's usage with component creation?

Reply:
AD. 2. This is only the assumption, but fair enough for now. 
> If we decide to set width in content we will need to calculate for every case manually from designs.
> I propose to set implicitWidth and implicitHeight in Popup's content item and override width in StatusModal like width: content.implicitWidth + padding * 2

Calculations are not needed, `width: undefined` should be enough.

AD. 3. What is `destroy()`? I can't see that slot in qml Popup (StatusModal) :thinking:

Reply:
fixed in new commit